Tests/ResilientDecodingTests/ResilientRawRepresentableDictionaryTests.swift

```diff
@@ -146,4 +146,29 @@ final class ResilientRawRepresentableDictionaryTests: XCTestCase {
     #endif
   }
 
+  /**
+   When a `Dictionary` is decoded, the keys are not transformed by the `keyDecodingStrategy`.
+   */
+  func testKeyDecodingStrategyIsIgnored() throws {
+    let decoder = JSONDecoder()
+    decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Just to be clear it would be really weird for this to convert the keys right? I assume we're being safe (which I like!)

Reply:
This is the same behavior as a non-`Resilient` dictionary